### ISSUE TITLE
sygic call doesn't work on ios

### ISSUE THREAD

**Issue by Benoit-ROBIN** [dpa99c/phonegap-launch-navigator#83]
Hi,

Plugin works well on android but on iphone 6 ios 9.2 when I select Sygic's app the plugin call my callbackSuccess but nopthing happened. 
On Android it works well.

Did I misse something ?? It only happened on Sygic app.

**Comment by dpa99c** [comment]
Just tested this and it seems that the URL scheme for Sygic app on iOS has stopped working. When it was added to the plugin under issue #65, I tested it and it worked just fine at that time, so wondering if a recent update to the Sygic app has broken it.  [This stackoverflow question](http://stackoverflow.com/questions/28074766/strange-behaviour-of-custom-schema-for-sygic-on-ios) suggests it isn't the first time this has happened.

I can see in the Xcode console output that the URL scheme being generated by the plugin matches that [specified by Sygic](https://developers.sygic.com/documentation.php?action=customurl_ios).

I will investigate...
